src/test/scala/uk/gov/hmrc/hmrcfrontend/views/viewmodels/notificationbadge/NotificationBadgeSpec.scala

```diff
@@ -16,7 +16,39 @@
 
 package uk.gov.hmrc.hmrcfrontend.views.viewmodels.notificationbadge
 
+import play.api.libs.json.Json
 import uk.gov.hmrc.hmrcfrontend.views.viewmodels.JsonRoundtripSpec
 import uk.gov.hmrc.hmrcfrontend.views.viewmodels.notificationbadge.Generators._
 
-class NotificationBadgeSpec extends JsonRoundtripSpec[NotificationBadge]
+class NotificationBadgeSpec extends JsonRoundtripSpec[NotificationBadge] {
+  "NotificationBadge" should {
+    "accept strings" in {
+      val testData = NotificationBadge("3")
+
+      val json = Json.toJson[NotificationBadge](testData)
+
+      json.as[NotificationBadge] should be(
+        NotificationBadge("3"))
+    }
+    "accept integers" in {
+      val testData = NotificationBadge(1)
```

Comment:
I'm not sure whether this test really proves anything, given the integer 1 will be implicitly converted to a string.